app/views/collections/_sidebar.html.erb

```diff
@@ -1,62 +1,60 @@
 <div id="dashboard" class="region" role="navigation region">
   <h3 class="landmark heading">Dashboard</h3>
   <ul class="navigation actions">
-    <li><%= span_if_current ts('Dashboard', key: 'dashboard'), @collection %></li>
-    <li><%= span_if_current ts('Profile', key: 'dashboard'), collection_profile_path(@collection) %></li>
+    <li><%= span_if_current t(".dashboard"), @collection %></li>
+    <li><%= span_if_current t(".profile"), collection_profile_path(@collection) %></li>
     <% if !@collection.collection_profile.faq.blank? %>
-      <li><%= span_if_current ts('FAQ', key: 'dashboard'), collection_profile_path(@collection, :anchor => "faq") %></li>
+      <li><%= span_if_current t(".faq"), collection_profile_path(@collection, anchor: "faq") %></li>
     <% end %>
     <% if !@collection.collection_profile.rules.blank? %>
-      <li><%= span_if_current ts('Rules', key: 'dashboard'), collection_profile_path(@collection, :anchor => "rules") %></li>
+      <li><%= span_if_current t(".rules"), collection_profile_path(@collection, anchor: "rules") %></li>
     <% end %>
-  
+
     <% if @collection.parent %>
-      <li><%= link_to ts('Parent Collection', key: 'dashboard'), collection_path(@collection.parent) %></li>
+      <li><%= link_to t(".parent_collection"), collection_path(@collection.parent) %></li>
     <% else %>
-      <li><%= span_if_current ts("Subcollections (%{count})", key: 'dashboard', :count => @collection.children.count), collection_collections_path(@collection) %></li>
+      <li><%= span_if_current t(".subcollections", count: @collection.children.count), collection_collections_path(@collection) %></li>
     <% end %>
   </ul>
 
   <!-- challenge section of dash -->
   <% if @collection.challenge %>
-    <%= render :partial => "challenge/#{challenge_class_name(@collection)}/challenge_sidebar" %>
+    <%= render partial: "challenge/#{challenge_class_name(@collection)}/challenge_sidebar" %>
   <% end %>
 
   <h4 class="landmark heading">Contents</h4>
```

Comment:
Please make this text translatable as well, I would suggest `t(".landmark.contents")`.